OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -5,7 +5,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Optimize>true</Optimize>
     <UseVSHostingProcess>false</UseVSHostingProcess>
-    <LangVersion>5</LangVersion>
+    <LangVersion>6</LangVersion>
```

Comment:
@OpenRA/core I suspect we aren't ready for this language version bump

Reply:
While it doesn't seem to effect the .NET framework version for end users. According to https://docs.microsoft.com/dotnet/csharp/language-reference/compiler-options/langversion-compiler-option C# 6.0 is an unofficial draft at the moment. We shouldn't make use of it as of now.

Reply:
Old habits here, @pchote did raise this in the other PR I was working on, which I fixed and downgraded back to 5. Will do the same here, and for the other related items.